components/ingest-service/test/liveness.js

```diff
@@ -27,6 +27,7 @@ describe("/events/chef/liveness", function () {
   });
 
   describe("posting example message", function () {
+    before(function () { return helpers.ESCleanupDocuments() });
```

Comment:
I am not sure why this is needed now, but without this line there is a node left in the node-state index with the id "test-id".